torch/csrc/autograd/init.cpp

```diff
@@ -228,6 +228,15 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject* unused) {
                 });
             return as_pyobj;
           })
+      .def(
+          "kwinputs",
```

Comment:
Should we use the profiler/python/init.cpp to bind these profiler APIs?

Reply:
good idea, maybe we can do this in a separate PR though since it would involve moving all the KinetoEvent methods?

Reply:
Agreed, can follow up